File: src/main/java/com/haozileung/web/service/logging/IOperateLogService.java
package com.haozileung.web.service.logging;

import com.haozileung.infra.pager.PageResult;
import com.haozileung.web.domain.logging.OperateLog;

public interface IOperateLogService {

    void save(OperateLog dictionary);

    void update(OperateLog dictionary);

    void delete(OperateLog dictionary);

    PageResult<OperateLog> query(OperateLog dictionary, int pageNo, int pageSize);
}

File: src/main/kotlin/com/haozileung/web/service/logging/IOperateLogService.kt
package com.haozileung.web.service.logging

import com.haozileung.infra.pager.PageResult
import com.haozileung.web.domain.logging.OperateLog

interface IOperateLogService {

    fun save(dictionary: OperateLog)

    fun update(dictionary: OperateLog)

    fun delete(dictionary: OperateLog)

    fun query(dictionary: OperateLog, pageNo: Int, pageSize: Int): PageResult<OperateLog>
}
